skimage/util/noise.py

```diff
@@ -5,6 +5,36 @@
 __all__ = ['random_noise']
 
 
+def choose(p, shape):
+    """
+    Function to return Bernoulli trials at a given probability
```

Comment:
You can write "Bernoulli trials at a given probability of a given size." on a single line